packages/protocol/test/identity/attestations.ts

```diff
@@ -95,27 +95,24 @@ contract('Attestations', (accounts: string[]) => {
     registry = await Registry.new(true)
     await accountsInstance.initialize(registry.address)
     await registry.setAddressFor(CeloContractName.Validators, mockValidators.address)
-
     const tokenBalance = web3.utils.toWei('10', 'ether').toString()
-    await Promise.all(
-      accounts.map(async (account) => {
-        await mockERC20Token.mint(account, tokenBalance)
-        await otherMockERC20Token.mint(account, tokenBalance)
-        await accountsInstance.createAccount({ from: account })
-        await unlockAndAuthorizeKey(
-          KeyOffsets.VALIDATING_KEY_OFFSET,
-          accountsInstance.authorizeValidatorSigner,
-          account,
-          accounts
-        )
-        await unlockAndAuthorizeKey(
-          KeyOffsets.ATTESTING_KEY_OFFSET,
-          accountsInstance.authorizeAttestationSigner,
-          account,
-          accounts
-        )
-      })
-    )
+    for (const account of accounts) {
```

Comment:
Switched to `for loop` due to test timming-out.

This could be related to transactions not beeing received in the order submitted.